Specs/Cesium3DTilesSpecHelper.js

```diff
@@ -0,0 +1,262 @@
+/*global define*/
+define([
+        'Core/Color',
+        'Core/defaultValue',
+        'Core/OrientedBoundingBox',
+        'Scene/Cesium3DTileContentProviderFactory',
+        'Scene/Cesium3DTileContentState',
+        'Scene/Cesium3DTileset',
+        'Specs/pollToPromise'
+    ], function(
+        Color,
+        defaultValue,
+        OrientedBoundingBox,
+        Cesium3DTileContentProviderFactory,
+        Cesium3DTileContentState,
+        Cesium3DTileset,
+        pollToPromise) {
+    "use strict";
+
+    var Cesium3DTilesSpecHelper = function() {
+    };
+
+    function verifyRender(scene, tileset) {
+        tileset.show = false;
+        expect(scene.renderForSpecs()).toEqual([0, 0, 0, 255]);
+        tileset.show = true;
+        var pixelColor = scene.renderForSpecs();
+        expect(pixelColor).not.toEqual([0, 0, 0, 255]);
+        return pixelColor;
+    }
+
+    function verifyRenderBlank(scene, tileset) {
+        tileset.show = false;
+        expect(scene.renderForSpecs()).toEqual([0, 0, 0, 255]);
+        tileset.show = true;
+        expect(scene.renderForSpecs()).toEqual([0, 0, 0, 255]);
+    }
+
+    Cesium3DTilesSpecHelper.verifyRenderTileset = function(scene, tileset) {
```

Comment:
Perhaps we should rename these functions with calls to `expect` to something like `expectRenderTileset ` (I know `verify` is used in old tests).